test/Conversion/tosa-to-emitc.mlir

```diff
@@ -126,7 +126,7 @@ func @test_sub(%arg0: tensor<13x21x1xf32>, %arg1: tensor<13x21x3xf32>) -> tensor
 // Other ops
 
 func @test_conv2d(%arg0: tensor<1x4x4x4xf32>, %arg1: tensor<8x1x1x4xf32>, %arg2: tensor<8xf32>) -> tensor<1x4x4x8xf32> {
-    // CHECK: %0 = emitc.call "tosa::conv2d"(%arg0, %arg1) {args = [0 : index, 1 : index, [0, 0, 0, 0], [1, 1], [1, 1]], template_args = [tensor<1x4x4x8xf32>]} : (tensor<1x4x4x4xf32>, tensor<8x1x1x4xf32>) -> tensor<1x4x4x8xf32>
+    // CHECK: %0 = emitc.call "tosa::conv2d"(%arg0, %arg1) {args = [0 : index, 1 : index, dense<0> : tensor<4xi64>, dense<1> : tensor<2xi64>, dense<1> : tensor<2xi64>], template_args = [tensor<1x4x4x8xf32>]} : (tensor<1x4x4x4xf32>, tensor<8x1x1x4xf32>) -> tensor<1x4x4x8xf32>
```

Comment:
`dense<0> : tensor<4xi64>` "expands" logically to `dense<[0, 0, 0, 0]> : tensor<4xi64>` which would be more explicit, but I think this is fine like it is.

Reply:
`dense<[0, 0, 0, 0]> : tensor<4xi64>` won't work, because I don't know how to force MLIR to keep the array whole. It just drops all the elements :man_shrugging:

Reply:
We have to go with what mlir prints by default, don't we? :)